packages/extension-collaboration-cursor/package.json

```diff
@@ -28,14 +28,13 @@
     "src",
     "dist"
   ],
-  "dependencies": {
-    "y-prosemirror": "1.0.20"
-  },
   "peerDependencies": {
-    "@tiptap/core": "^2.0.0-beta.209"
+    "@tiptap/core": "^2.0.0-beta.209",
+    "y-prosemirror": "1.0.20"
   },
   "devDependencies": {
-    "@tiptap/core": "^2.0.0-beta.218"
+    "@tiptap/core": "^2.0.0-beta.218",
+    "y-prosemirror": "1.0.20"
```

Comment:
Why move this to peer&dev dependencies?   I think it should be a dependency of the extension itself . Now i have to manually depend on it in the main project after upgrading.